Modules/IO/GDCM/src/itkGDCMImageIO.cxx

```diff
@@ -188,8 +188,6 @@ readNoPreambleDicom(std::ifstream & file) // NOTE: This file is duplicated in it
     }
   } while (groupNo == 2);
 
-  itkDebugMacro(<< "No DICOM magic number found, but the file appears to be DICOM without a preamble.");
```

Comment:
Thanks Matt! 

- By the way, I did already hesitate to comment about the use of `<<`, because we have agreed before that it isn't really necessary for such `itkDebugMacro` calls: https://github.com/InsightSoftwareConsortium/ITK/pull/4059 commit b52f423117bac0a0982c58960f7a80e1a25e6f2b